cni/pkg/cmd/root.go

```diff
@@ -100,11 +102,22 @@ var rootCmd = &cobra.Command{
 			// as well as listen for messages from the CNI binary.
 			cniEventAddr := filepath.Join(cfg.InstallConfig.CNIAgentRunDir, constants.CNIEventSocketName)
 			log.Infof("Starting ambient node agent with inpod redirect mode on socket %s", cniEventAddr)
+
+			// instantiate and validate the ambient enablement selector
+			selectors := []util.EnablementSelector{}
+			if err = yaml.Unmarshal([]byte(cfg.InstallConfig.AmbientEnablementSelector), &selectors); err != nil {
```

Comment:
Confused - is this yaml in yaml ?Is this a new pattern ?

Reply:
it's actually yaml in an env var.